packages/@okta/vuepress-site/docs/guides/email-customization/customize-email-templates/index.md

```diff
@@ -28,6 +28,105 @@ Email templates use common and unique [Expression Language (EL) variables](https
 
 > **Note:** Some templates listed in the [variables tables](https://help.okta.com/en/prod/okta_help_CSH.htm#ext-expression-language) may not appear in your org. To obtain these templates, contact [Okta Support](https://support.okta.com/help/s/?_ga=2.17747641.1660906902.1597076228-1076744453.1575496867).
 
+### Customize your org's templates by using Velocity Templating Language (VTL)
+<ApiLifecycle access="ea" />
+
+#### Overview of what this EA feature enables
+
+This Early Access (EA) feature provides customers with the functionality to customize the email templates, which Okta provides, that allows them to send email notifications to their end users. It changes the underlying email templating engine to Velocity so that you can use the syntax of the Velocity Templating Language (VTL) to customize your org's email templates. In addition to the new syntax, which provides enhanced conditional logic, some org attributes and access are provided to all attributes in the Okta [User Profile object](/docs/reference/api/users/#profile-object).
```

Comment:
The first reference of Velocity should say "Velocity Templating Language (VTL)" and subsequent ones, maybe refer to it as Velocity macros? or just "Enhanaced email macros"? I'm wondering how much we should even call out VTL.

Reply:
Agree with this, maybe just keep the mention in the velocity syntax section below.  I didn't think Velocity was the draw to customers. They need to know about the syntax but it maybe isn't a selling point worth mentioning in the headline

Reply:
Suggestion - link to `https://velocity.apache.org/` on the first reference to VTL.

IMHO, since we are providing VTL as is, we should stay as close as possible to the Velocity terminology (e.g., variables and properties instead of macros) and link to the Velocity docs where it makes sense.

Reply:
Fixed sentence. 
Updated "syntax" references to "Velocity syntax".
Added link on first reference to VTL.